### DISCUSSION TITLE
Opensource Client of mRemoteNG for Macos and Linux

### DISCUSSION THREAD

**Discussion by omid1979** [mRemoteNG/mRemoteNG#2501] [Ideas]
hi all , 
the mRemoteNG is very usefull application for me to use on my office computer to connect out server , 
It is good idea to make it availble for linux and macos to enable macos users to use this app to connect servers , 
is it possible ?

**Comment by omid1979** [comment]
there is no centeralized free and opensource application on macos to enable connect to all windows and linux and also telnet service from single application ,

**Reply by dstejskal88** [reply]
Actually there is an alternative. Try [CENSORED](https://cdn.pixabay.com/photo/2021/12/08/23/50/censored-6856902_960_720.png), which is free for individual users. There is also support for import of configurations from mRemoteNG.

**Reply by Kvarkas** [reply]
Technicaly I dont mind if users using another software but I am agains free promotions on my solution pages, they dont pay me for such so why should I let it?

**Comment by savornicesei** [comment]
It's not that easy to make mRemoteNG cross-platform as it depends on several Windows-only components (like Putty) and it's UI needs to be reworked to use AvaloniaUI (if targeting .NET core).

**Reply by igor2x** [reply]
Putty is not Windows-only, at least on Linux it exists for ages, like at least for a decade.

**Comment by Kvarkas** [comment]
At the moment its not posible, but in feature (once can get to it) its a plan separate UI and add modularity, so any terminal could be included to operate, thats mean that it may be easy to port it to Linux and Mac as well, but its will be not soon if no support on such :( at the moment I am support a code base almost alone (thanks for few fellows who submiting PRs) but that seriouse job and I dont have enoght expirience at the moment to promiss it soon.

**Reply by nicthegarden** [reply]
That would be awesome to have as a unix / linux sysadmin perspective. 

Thansk

**Comment by jijtech** [comment]
+1 for my interest in a linux client, my company has been using MRemoteNG for years religiously, but if no linux client comes out, then we gonna have to part ways with it since we're switching to linux of EU origin to follow GDPR

**Reply by Kvarkas** [reply]
1.8 x expect to be multy platform, but that for feature plans, at this moment i have very little knowleadge of how to code multiplatforms apps :(
